src/cms/templates/users/welcome_email.html

```diff
@@ -0,0 +1,40 @@
+{% load i18n %}
+{% autoescape off %}
+
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+</head>
+<body>
+    <div style="text-align:justify; font-family:sans-serif; background-color: rgba(241, 245, 249, 100); padding: 1rem; border-radius: 1rem; line-height:1.5">
+        <img src='cid:logo' style="width:200px;"> 
```

Comment:
```suggestion
        <img src='cid:logo' alt="" style="width:200px;"> 
```
I think you need to pass an empty alt text if you want the image to be invisible if it can't be loaded... Otherwise, there will be a placeholder that indicates a broken image.